source/API_Reference/Web_API_v3/Marketing_Campaigns/contactdb.apiblueprint

```diff
@@ -779,17 +805,10 @@ Mon, 02 Feb 2015 23:59:59 GMT.
 
 # Group Segments
 
-Valid operators for create and update depend on the type of the field you are segmenting.
-<ul>
-<li>Dates: "eq", "ne", "lt" (before), "gt" (after)</li>
-<li>Text: "contains", "eq" (is - matches the full field), "ne" (is not - matches any field where the entire field is not the condition value)</li>
-<li>Numbers: "eq", "lt", "gt"</li>
-<li>Email Clicks and Opens: "eq" (opened), "ne" (not opened)</li>
-</ul>
-
-Segment conditions using "eq" or "ne" for email clicks and opens should provide a "field" of either clicks.campaign_identifier or opens.campaign_identifier. The condition value should be a string containing the id of a completed campaign.
-
-Segments may contain multiple condtions, joined by an "and" or "or" in the "and_or" field. The first condition in the conditions list must have an empty "and_or", and subsequent conditions must all specify an "and_or".
+<p>
+  Body contains a JSON object with <code>conditions</code>, a list of conditions as described below, a unique <code>name</code>, and an optional <code>list_id</code>, which is a valid list ID for a list to limit the search on.
+  The JSON accepted by this endpoint is identical to that of the POST search with conditions, with the addition of requiring a unique name.  (POST search with conditions is described above.)
```

Comment:
Instead of redundant accepted params.  I said this is the same as post with search conditions